echo-pipelinetriggers/echo-pipelinetriggers.gradle

```diff
@@ -27,4 +27,8 @@ dependencies {
   compile spinnaker.dependency("okHttp")
   compile spinnaker.dependency("eurekaClient")
   compile spinnaker.dependency("kork")
+
+  compile group: 'org.codehaus.groovy', name: 'groovy', version: "2.4.5"
```

Comment:
Should be able to use `compile spinnaker.dependency('groovy')`and `spinnaker.group('spockBase')` vs. explicit dependencies.